From 95531b608ad20959590cb044552d5e7faacc6828 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Mon, 6 Feb 2023 10:14:23 +0800
Subject: [PATCH 18/45] Revert "drm/i915/dsi: fix VBT send packet port
 selection for dual link DSI"

This reverts commit bf83a303f2b56fc996ee9afd53a0a2cc71ffffad.
---
 drivers/gpu/drm/i915/display/intel_dsi_vbt.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/display/intel_dsi_vbt.c b/drivers/gpu/drm/i915/display/intel_dsi_vbt.c
index 117ea44fd8be7..8777d35ac7a73 100644
--- a/drivers/gpu/drm/i915/display/intel_dsi_vbt.c
+++ b/drivers/gpu/drm/i915/display/intel_dsi_vbt.c
@@ -133,9 +133,9 @@ static enum port intel_dsi_seq_port_to_port(struct intel_dsi *intel_dsi,
 		return ffs(intel_dsi->ports) - 1;
 
 	if (seq_port) {
-		if (intel_dsi->ports & BIT(PORT_B))
+		if (intel_dsi->ports & PORT_B)
 			return PORT_B;
-		else if (intel_dsi->ports & BIT(PORT_C))
+		else if (intel_dsi->ports & PORT_C)
 			return PORT_C;
 	}
 
-- 
2.25.1

